examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,148 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide your API Key, keep this secure
+lob.api_key = 'test_<your key here>'
+###########################################################
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python letter.py <CSV_FILE>")
+    sys.exit(1)
+
+# Open input files
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'name',
+    'id',
+    'url',
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+# create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: ' + str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+with open('letter.html', 'r') as html_file:
+    letter_html = html_file.read()
+
+with open(input_filename, 'r') as input, \
+        open(success_filename, 'w') as success, \
+        open(errors_filename, 'w') as errors:
+
+    # Print mode to screen
+    mode = lob.api_key.split('_')[0]
+    print('Sending letters in ' + mode.upper() + ' mode.')
+
+    input_csv = csv.DictReader(input)
+    errors_csv_fields += input_csv.fieldnames
+
+    success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
+    errors_csv = csv.DictWriter(errors, fieldnames=errors_csv_fields)
+
+    # Create from_address
+    try:
+        from_address = lob.Address.create(
+            name='Your Name/Company',
```

Comment:
What about adding all these configs at the top together the `api_key` such that a user will only have to modify the top part of the file. We can have a dictionary holding all these values.